xbmc/cores/RetroPlayer/cheevos/Cheevos.cpp

```diff
@@ -29,6 +33,11 @@ constexpr auto SUCCESS = "Success";
 constexpr auto GAME_ID = "GameID";
 constexpr auto PATCH_DATA = "PatchData";
 constexpr auto RICH_PRESENCE = "RichPresencePatch";
+constexpr auto GAME_TITLE = "Title";
+constexpr auto PUBLISHER = "Publisher";
+constexpr auto DEVELOPER = "Developer";
+constexpr auto GENRE = "Genre";
+constexpr auto CONSOLE_NAME = "ConsoleName";
 
 constexpr int RESPORNSE_SIZE = 64;
```

Comment:
Absolutely not related to this, but stuck out at me. Typo

Reply:
Pushed a fix